tekton/ci/jobs/tekton-github-tasks-completed.yaml

```diff
@@ -0,0 +1,27 @@
+apiVersion: tekton.dev/v1beta1
+kind: Pipeline
+metadata:
+  name: tekton-github-tasks-completed
+  namespace: tektonci
+spec:
+  params:
+    - name: checkName
+      description: The name of the GitHub check that this pipeline is used for
+    - name: gitHubCommand
+      description: The command that was used to trigger testing
+    - name: body
+      description: The body of the Pull Request
+  tasks:
+    - name: github-tasks-completed
+      conditions:
+        - conditionRef: "check-name-matches"
+          params:
+            - name: gitHubCommand
+              value: $(params.gitHubCommand)
+            - name: checkName
+              value: $(params.checkName)
```

Comment:
since the above comments, `ci-workspaces` has been removed and `ci` has been updated -- rebased and updated the implementation in this pr as well (including using `when` expressions)

thanks @bobcatfish!